server/lib/payments.js

```diff
@@ -914,11 +914,15 @@ export const getHostFeeSharePercent = async (order, host = null) => {
   const possibleValues = [];
 
   if (order) {
-    if (order.paymentMethod?.service === 'stripe' && order.paymentMethod?.type === 'creditcard') {
-      possibleValues.push(plan?.creditCardHostFeeSharePercent);
+    // Preload payment method
```

Comment:
That's the right approach.

We need to be defensive and can't rely on the property being set.